phys/module_bl_mynn.F

```diff
@@ -2151,6 +2151,7 @@ SUBROUTINE  mym_condensation (kts,kte,  &
     zagl = 0.
 ! Note: kte needs to be larger than kts, i.e., kte >= kts+1.
     DO k = kts,kte-1
+       p2a = exner(k)
```

Comment:
@ahijevyc 
Dave,
Take a look at
https://github.com/wrf-model/WRF/blob/master/phys/module_bl_mynn.F
Does the problem look like it has been corrected?

Reply:
Yes - removing the variable 'p2a' entirely and replacing with exner(k) also fixes the issue. 
Looks like this is what was done.

Reply:
Ming Chen sent me an e-mail on April 24 following up on when I reported this issue, and in Friendly Beta 3 it is fixed. His "official" fix bypasses p2a and just uses exner directly, as suggested by ahijevyc.

Reply:
Thanks for your quick responses.  I'm pretty new at this.    I've only made a couple pull requests, ever. 
 Is there anything else I should do to at this time, like withdraw my pull request or pull this file from the private repo in the above link?